[Hugo] Add archetype for tutorial template (#14897)
cloudflare/cloudflare-docs: content/style-guide/documentation-content-strategy/content-types/tutorial.md

# Tutorial

A tutorial connects products to real-world scenarios to meet a user's goal.

## Template

> We have a [pre-built template](https://github.com/cloudflare/cloudflare-docs/tree/production/archetypes/tutorial.md) that helps you follow our general structure and content guidelines.

Reviewer: ```suggestion
We have a [pre-built template](https://github.com/cloudflare/cloudflare-docs/blob/production/archetypes/tutorial.md) that helps you follow our general structure and content guidelines.
```
For files I believe it's `blob`.

+ A dumping ground for screenshots
+ Content with no end goal or job-to-be-done

### Tone

Guiding, straightforward, educational, authoritative

### content_type

`tutorial`

### Structure

#### Required components

[**Title**](/style-guide/documentation-content-strategy/component-attributes/titles/): Short verb phrase in second-person imperative.

> **Context**: An introductory paragraph on the user's goal or job-to-be-done and how they'll accomplish that in the tutorial. Consider including the intended audience for the tutorial. Refer to [Context](/style-guide/documentation-content-strategy/component-attributes/context/) for more information.

Reviewer: ```suggestion
**Context**: An introductory paragraph on the user's goal or job-to-be-done and how they will accomplish that in the tutorial. Consider including the intended audience for the tutorial. Refer to [Context](/style-guide/documentation-content-strategy/component-attributes/context/) for more information.
```

**Consider the user story framing**: "As a `___`, I want to `___` so I can `___`."

> [**Steps**](/style-guide/documentation-content-strategy/component-attributes/next-steps/): Numbered steps that complete a task. Refer to [Steps/tasks/procedures](/style-guide/documentation-content-strategy/component-attributes/steps-tasks-procedures/) for more information.

Reviewer: ```suggestion
[**Steps**](/style-guide/documentation-content-strategy/component-attributes/steps-tasks-procedures/): Numbered steps that complete a task.
```
Link was incorrect, but appeared correct later. Now it was duplicated, so I think it's better to remove the last sentence.
